### PR TITLE
Portfolio Project

### PR DESCRIPTION
**Pull request by karam084**
- Create index.html include on ( Header, Selections, Logo and Menu hamburger).
- Create style.css include on ( styles properties, flexbox display, and figma templet positions.
- Add GitHub linter file.
- Create a Source file.
-Create a media file.
- Create a Screenshot file for project phases.
- Create image cons for the project background.
- Add Copyright the permission file from Figma.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        <meta name=viewport content="width=device-width, initial-scale=1">
+     </head>
+     <body>
+         <!--header -->
+         <header class ="header">
+             <div class="img_icon"></div>
+             <div class="time_icon"></div>
+             <div class="cellular_con"></div>
+             <div class="wifi_path"></div>
+             <div class="capacity"></div>
```
Comment (Alexoid1) [inline index.html]: - [x] Kindly delete those containers. The toolbar always is part of the phone itself.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+             <a class="logo" href="#" target="_blank">My Logo</a>
+                 <!-- hum_group-->
+             <div class="nav-elem">
```
Comment (Alexoid1) [inline index.html]: - [x] Kindly replace that `div` with a `nav` tag. ln order to add SEO support to the website.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+         <!--section 3 more details about me-->
+         <section class="box flex-container">
+             <h1 class="my_name">I’m Karam Glad to see you!</h1>
+             <p class="my_details">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+             <h2 class="more_det">LET’S CONNECT</h2>
+         </section>
+         <!--media icons-->
+         <div class="media_div">
+             <div class="media_ul">
+                 <a class="twitter" href="https://twitter.com/ElarabFouad" target="_blank" rel="noopener noreferrer"><img src="source/media/twitter@2x.png" alt="twitter"/></a>
```
Comment (Alexoid1) [inline index.html]: - [x] You need to fix the styles for the mobile version. Make sure to use the values of `font family, font size, color, margin and font-weight` same as Figma template.
Captura de pantalla_2021-09-21_21-38-12

**Figma**
Captura de pantalla_2021-09-21_21-39-05

- [OPTIONAL] Kindly read this article in order to learn how to use Chrome Dev Tools to [Simulate mobile devices with Device Mode](https://developer.chrome.com/docs/devtools/device-mode/).

File: index.html [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+             </div>
+         </header>
+         <!--section 3 more details about me-->
+         <section class="box flex-container">
+             <h1 class="my_name">I’m Karam Glad to see you!</h1>
+             <p class="my_details">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+             <h2 class="more_det">LET’S CONNECT</h2>
+         </section>
+         <!--media icons-->
+         <div class="media_div">
```
Comment (Alexoid1) [inline index.html]: - [ ] Kindly replace div with a ul tag and li tags to list the icons.

**Review by Alexoid1 — CHANGES_REQUESTED** [review]
# Hi @karam084,

Impressive project!
You are doing a great job so far, and the project is looking good.

## Required Changes :recycle:
 For the project to get approved implement the changes mentioned below.

## Optional suggestions
- [OPTIONAL] Kindly read this article in order to learn how to use Chrome Dev Tools to [Simulate mobile devices with Device Mode](https://developer.chrome.com/docs/devtools/device-mode/).

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED ✔🎉**
Hello @karam084 👋🏽
Your project is complete!  There is nothing else to say. I's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by karam084** [comment]
> **STATUS: APPROVED ✔🎉**
> Hello @karam084 👋🏽
> Your project is complete! There is nothing else to say. I's time to merge it :shipit:
> Congratulations! 🎉
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Oh, this is great news, thank you very much.

**Pull request merged**